integration/simple_app_test.go

```diff
@@ -65,16 +69,48 @@ func testSimpleApp(t *testing.T, context spec.G, it spec.S) {
 				Execute(name, source)
 			Expect(err).NotTo(HaveOccurred())
 
-			container, err = docker.Container.Run.
-				WithCommand(fmt.Sprintf("ls -alR /layers/%s/modules/node_modules", strings.ReplaceAll(buildpackInfo.Buildpack.ID, "/", "_"))).
+			// check the contents of the node modules
+			container1, err = docker.Container.Run.
+				WithCommand(fmt.Sprintf("ls -alR /layers/%s/modules/node_modules",
+					strings.ReplaceAll(buildpackInfo.Buildpack.ID, "/", "_"))).
 				Execute(image.ID)
 			Expect(err).NotTo(HaveOccurred())
 
 			Eventually(func() string {
-				cLogs, err := docker.Container.Logs.Execute(container.ID)
+				cLogs, err := docker.Container.Logs.Execute(container1.ID)
 				Expect(err).NotTo(HaveOccurred())
 				return cLogs.String()
 			}).Should(ContainSubstring("leftpad"))
+
+			// check that all expected SBOM files are present
+			container2, err = docker.Container.Run.
+				WithCommand(fmt.Sprintf("ls -alR /layers/sbom/launch/%s/modules/",
```

Comment:
@sophiewigmore if we care about the exact directory of the SBOM files (vs that they live under some hierarchy) then we should use `ls -al` rather than `ls -alR`

Reply:
done in https://github.com/paketo-buildpacks/yarn-install/pull/285/commits/50f481482957472166295f610c3f7060590592a7